api/src/main/java/org/openmrs/validator/ValidateUtil.java

```diff
@@ -146,4 +149,31 @@ public static void setDisableValidation(Boolean disableValidation) {
 		ValidateUtil.disableValidation = disableValidation;
 	}
 
+	/**
+	 * @return true if validation has been disabled for the current thread, false otherwise
+	 * @since 2.5.8
+	 */
+	public static boolean isValidationDisabledForThread() {
+		return disableValidationForThread.get() == Boolean.TRUE;
+	}
+
+	/**
+	 * Sets a ThreadLocal variable to indicate that validation should be disabled for the current thread
+	 * NOTE: This should always be used in conjunction with the resumeValidationForThread method to 
```

Comment:
Just a minor comment, i would not include the implementation details `Sets a ThreadLocal variable` in the method description.

Reply:
Changed